daemon/daemon.go

```diff
@@ -74,7 +74,10 @@ import (
 )
 
 // ContainersNamespace is the name of the namespace used for users containers
-const ContainersNamespace = "moby"
+const (
+	isWindows           = runtime.GOOS == "windows"
```

Comment:
you could just set this in daemon_windows.go and daemon_unix.go as constants and not import `runtime` at all, its kind of even more constanty.

Reply:
ah, good point we have separate files for both; yes, I'll update

Reply:
updated 👍